components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -3163,11 +3163,20 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
      */
     protected async getImageBuilderClient(user: User, workspace: Workspace, instance?: WorkspaceInstance) {
         const teams = await this.teamDB.findTeamsByUser(user.id);
-        const isMovedImageBuilder = await getExperimentsClientForBackend().getValueAsync("movedImageBuilder", false, {
-            user,
-            projectId: workspace.projectId,
-            teams,
-        });
+        let isMovedImageBuilder = false;
+        for (const team of teams) {
```

Comment:
This might be one of the cases where teams is useful But also, we don't use it, yet. And we can also use the "Gitpod" segement.
So let's remove as well.